### PR TITLE
Nginx config mw

### PR DESCRIPTION
**Pull request by mwolff44**
Depends-On: https://github.com/wazo-platform/sf-jobs/pull/51

### CODE REVIEW

**Comment by sboily** [comment]
Can you use https instead http please.

File: debian/wazo-router-confd.dirs [hunk 1]
```diff
@@ -0,0 +1 @@
+etc/nginx/locations/http-enabled
```
Comment (sboily) [inline debian/wazo-router-confd.dirs]: https

File: etc/nginx/locations/http-available/wazo-router-confd [hunk 2]
```diff
@@ -0,0 +1,8 @@
+location ^~ /api/router_confd/ {
+    proxy_pass http://127.0.0.1:8000/;
+    # client_max_body_size 16m;
+
+    proxy_set_header    Host                $http_host;
+    proxy_set_header    X-Script-Name       /api/router_confd;
+    proxy_set_header    X-Forwarded-For     $proxy_add_x_forwarded_for;
+}
```
Comment (sboily) [inline etc/nginx/locations/http-available/wazo-router-confd]: Change the name of the directory to use https instead http.

Comment (mwolff44) [reply etc/nginx/locations/http-available/wazo-router-confd]: oups

Comment (mwolff44) [reply etc/nginx/locations/http-available/wazo-router-confd]: For https, i wanted for test to keep it without ssl

Comment (sboily) [reply etc/nginx/locations/http-available/wazo-router-confd]: Why? I'm not sure to understand. We don't expose any API by nginx in http. There is no reason to expose it in http only.

File: debian/wazo-router-confd.postinstall [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+        if [ ! -e "$LOG_FILENAME" ]; then
+            touch "$LOG_FILENAME"
+        fi
+
+        chown www-data: "$LOG_FILENAME"
+
+        if [[ -z "${previous_version}" ]]; then
+            ln -sf  /etc/nginx/locations/http-available/$DAEMONNAME \
+                    /etc/nginx/locations/http-enabled/$DAEMONNAME
```
Comment (sboily) [inline debian/wazo-router-confd.postinstall:22]: Please fix http to https.

File: etc/nginx/locations/https-available/wazo-router-confd [hunk 4]
```diff
@@ -0,0 +1,8 @@
+location ^~ /api/router_confd/ {
+    proxy_pass http://127.0.0.1:8000/;
```
Comment (sboily) [inline etc/nginx/locations/https-available/wazo-router-confd:2]: We need to use another port than 8000 because this is the default port.

**Review by sboily — CHANGES_REQUESTED** [review]

**Comment by tranchitella** [comment]
@mwolff44 @fredericlepied what's the status of this PR? Can we finalize and merge it?

**Review by tranchitella — CHANGES_REQUESTED** [review]
We have to support HTTPS as all the other wazo modules are HTTPS-based.
You also need to change the default port from 8000 to something else (without clashing the other wazo services).

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Pull request closed without merging**